pkg/commands/run.go

```diff
@@ -479,9 +479,9 @@ func (e *Executor) executeRun(_ *cobra.Command, args []string) {
 
 // to be removed when deadline is finally decommissioned
 func (e *Executor) setTimeoutToDeadlineIfOnlyDeadlineIsSet() {
-	// nolint:staticcheck
-	deadlineValue := e.cfg.Run.Deadline
+	deadlineValue := e.cfg.Run.Deadline // nolint
```

Comment:
this is weird, I'm not sure why `nolintnolint` fails on this.
@ldez do you have any ideas?

Reply:
It's a flaky behavior, increased/produced by the cache used by the golangci-lint GH action.
It's a real pain, I often have to relaunch the CI several times before I can merge a PR.

Reply:
pb partially fixed in #2359